static/app/types/breadcrumbs.tsx

```diff
@@ -35,6 +35,7 @@ interface BreadcrumbTypeBase {
   category?: string | null;
   event_id?: string | null;
   message?: string;
+  message_formatted?: string;
```

Comment:
@malwilley I wonder if it would be better to just use `message` here so the FE doesn't have to change. It's not like we're going to render `message` anywhere else, right?

Reply:
a breadcrumb is just a blog of text so I dont' think this is going to break anyone API usage

Reply:
I was thinking that it would useful for debugging when testing so we know what the original message was, otherwise it will be hard to know exactly what was changed by sqlparse.

I also thought we might at some point want a "formatted/raw" toggle like we do on the stack trace and CURL text. But this does come at the cost of sending a larger payload

Reply:
@malwilley If it's purely for debugging, then I'd do the reverse and make a `message_raw` field and just not even use it on the FE (just an API field).